src/components/Confirm/index.js

```diff
@@ -50,8 +50,8 @@ const Actions = ({retakeAction, confirmAction, error}) =>
 const Previews = localised(({capture, retakeAction, confirmAction, error, method, documentType, translate, isFullScreen}) => {
   const methodNamespace = method === 'face' ? `confirm.face.${capture.variant}` : `confirm.${method}`
   const title = translate(`${methodNamespace}.title`)
-  const altTag = translate(`${methodNamespace}.alt`)
-  const enlargedAltTag = translate(`${methodNamespace}.enlarged_alt`)
```

Comment:
Oh, i thought I removed this code along with the string but it seems I only removed the string from the locale file.  Good spot.